src/content/advanced/tls-certificates/index.md

```diff
@@ -0,0 +1,104 @@
+---
+linkTitle: TLS certificates with cert-manager
+title: Obtaining TLS certificates for Ingresses with cert-manager
+description: Configure cert-manager to automatically obtain TLS certificates for Ingresses
+weight: 20
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-cabbage
+user_questions:
+  - How do I install cert-manager?
+  - How do I obtain TLS certificates?
+last_review_date: 2022-07-19
+---
+
+Exposing HTTP services using an Ingress is a pretty straightforward task in Kubernetes. Using plain HTTP is, however, discouraged in favor of using HTTPS. `cert-manager` provides a simple, declarative way to automatically obtain TLS certificates for the endpoints we expose.
+
+Our App Platform provides a [`cert-manager-app`](https://github.com/giantswarm/cert-manager-app) that users can install. In order to configure it for TLS certificate management, we should install a `ClusterIssuer` that solves DNS challenges. Note that this also works for internal Load Balancers.
+
+We can do so by, first of all, disabling the default `ClusterIssuer` when installing the App, by providing the following user values:
+
+```yaml
+global:
+  giantSwarmClusterIssuer:
+    install: false
+```
+
+Then, we can create a new `ClusterIssuer` that solves DNS challenges. For instance, on Azure:
+
+```yaml
+apiVersion: cert-manager.io/v1
+kind: ClusterIssuer
+metadata:
+  name: letsencrypt-giantswarm
+spec:
+  acme:
+    server: https://acme-v02.api.letsencrypt.org/directory
+    email: test@customer.com
+    privateKeySecretRef:
+      name: letsencrypt-giantswarm
+    solvers:
+    - dns01:
+        azureDNS:
+          # find the right values in `azure.yaml` in the `external-dns` deployment
```

Comment:
I think this comment is a bit misleading. Most customers will know both their subscription ID and their cluster ID. So we should not point them to `kubectl exec` into `kube-system` pods.